dts/arm64/broadcom/viper-m7.dtsi

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Should this move, its cortex-m7?

Reply:
ah! good catch. The viper board is shipping an a72 and an m7 so we have some DTS duplicated. I copied the directory but forgot to clean it up.